geth/process.py

```diff
@@ -141,25 +160,25 @@ def wait_for_rpc(self, timeout=0):
                 _timeout.check()
 
     @property
-    def ipc_enabled(self):
+    def ipc_enabled(self) -> bool:
         return not self.geth_kwargs.get("ipc_disable", None)
 
     @property
-    def ipc_path(self):
+    def ipc_path(self) -> str:
         return self.geth_kwargs.get(
```

Comment:
And here too. I think it might look something like this:
```py
@property
def ipc_path(self) -> str:
    return getattr(self.geth_kwargs, "ipc_path", os.path.abspath(
        os.path.expanduser(
            os.path.join(
                self.geth_kwargs.data_dir,
                "geth.ipc",
            )
        )
    ))
```